DEVELOPERS.md

```diff
@@ -98,3 +98,66 @@ Each category lives in its own directory (for example `tests/unit`) and can be r
 Additional arguments passed to `just test` are passed on to pytest. For example, to
 run all the tests _except_ the functional tests, run `just test -k 'not functional'`,
 or to run a single test, run e.g. `just test tests/unit/test_urls.py::test_urls`.
+
+
+# Local job-server for integration.
+
+## First time set up
+
+This needs some first time setup, but after that is fairly simple to use. You
+will need the Bitwarden cli `bw` installed to pull the dev Github auth
+credentials. You need to run the following command with your github username.
+
+```
+just job-server/configure GHUSERNAME
```

Comment:
In addition to installing the bitwarden client, you also need to be logged in, otherwise `configure` asks for and accepts the master password but fails and says you're not logged in (but it's not entirely clear what you're not logged in to).

An extra note to also make sure you're logged in should be enough, I think? I guess the script could check if you're not already logged in and run this for you
```bw login --sso```